renovate.json5

```diff
@@ -8,7 +8,10 @@
   "timezone": "Asia/Kolkata",
   "ignoreDeps": [
     // Ignoring zxing update. Since update requires Java 8+ runtime.
-    "com.google.zxing:core"
+    "com.google.zxing:core",
+
+    // Ignoring this version because it introduces Material 3 specs
```

Comment:
Just a nitpick. Here we are saying "Ignoring this version" but we are not specifying any version. We should probably start considering having the version in the comment or see if there is an equivalent renovate config for version locking.